orttraining/orttraining/test/python/orttraining_test_onnxblock.py

```diff
@@ -537,11 +544,51 @@ def test_save_checkpoint():
     with tempfile.TemporaryDirectory() as checkpoint_dir_name:
         checkpoint_file_path = os.path.join(checkpoint_dir_name, "checkpoint")
         onnxblock.save_checkpoint((trainable_params, non_trainable_params), checkpoint_file_path)
-
         # Then
         assert os.path.exists(checkpoint_file_path)
 
 
+def test_load_checkpoint():
+    # Given
+    device = "cuda"
+    N, D_in, H, D_out = 64, 784, 500, 10
+    _, zero_onnx_model = _get_models(device, N, D_in, H, D_out, zero_flag=True)
```

Comment:
Can you update the variable names (see other tests that have updated the variable names).